msteams-platform/messaging-extensions/build-api-based-message-extension.md

```diff
@@ -45,26 +44,65 @@ Users must not enter a parameter for a header or cookie. If you need to pass hea
 
 <details><summary>2. App manifest</summary>
 
+* Set the app manifest version to `devPreview`.
 * Set `composeExtensions.composeExtensionType` to `apiBased`.
 * Define `composeExtensions.apiSpecificationFile` as the relative path to the OpenAPI Description file within the folder.
+* Define `apiSpecificationFile` as the relative path to the OpenAPI description document.
 * Define `apiResponseRenderingTemplateFile` as the relative path to the response rendering template.
 * Each command must have a link to the response rendering template.
 * Full description must not exceed 128 characters.
 * A command must have exactly one parameter.
+* Add a new optional property `authorization` under the `composeExtensions` node. This property should be null for bot-based ME and specified only for API ME.
+* Define the type of authentication your application uses by setting the `authType` property under the `authorization` node. The possible values for this property are `none`, `apiSecretServiceAuth`, `microsoftEntra`, and `userAuth`.
```

Comment:
We don't support userAuth. Please remove

Reply:
removed.